packages/lesmis/src/routes/AdminPanelApp.jsx

```diff
@@ -36,6 +36,15 @@ const PageContentContainerComponent = styled(PageContentWrapper)`
   }
 `;
 
+const AdminPanelWrapper = styled.div`
+  nav a {
+    font-size: 0.875rem; // make the font size smaller to fit more text in the nav and match the default font size
```

Comment:
Having a bit of trouble with LESMIS, so haven’t been able to verify this locally (sorry), but in theory this would do the trick:

```suggestion
    font-size: inherit;
```

Reply:
In this case I don't think so, because their theme is different and they have both regular LESMIS and admin panel, so in this case it's to confine the styling to the admin panel only